TelCo.ColorCoder/Program.cs

```diff
@@ -1,145 +1,40 @@
 ﻿using System;
 using System.Diagnostics;
 using System.Drawing;
+using TelCo.ColorCoder.Utils;
+using static TelCo.ColorCoder.Utils.Colors;
 
 namespace TelCo.ColorCoder
 {
-    /// <summary>
-    /// The 25-pair color code, originally known as even-count color code, 
-    /// is a color code used to identify individual conductors in twisted-pair 
-    /// wiring for telecommunications.
-    /// This class provides the color coding and 
-    /// mapping of pair number to color and color to pair number.
-    /// </summary>
     class Program
```

Comment:
Please introduce back all the comments. 
Always leave the code better than what it already was. 
Do not remove what is already good.
The comments in many cases are extracted to auto generate Documentation

Refer : https://docs.microsoft.com/en-us/visualstudio/ide/reference/generate-xml-documentation-comments?view=vs-2019

Reply:
added the comments back.